doc/nrf/device_guides/working_with_fem.rst

```diff
@@ -637,6 +670,17 @@ The *childImageName_* parameter can take the following values:
 *  ``802154_rpmsg_`` for applications with support for 802.15.4, but without support for Bluetooth
 *  ``hci_ipc_`` for application with support for Bluetooth, but without support for 802.15.4
 
+.. note::
+   On nRF53 devices, SPIM0 and UARTE0 are mutually exclusive AHB bus masters on the network core as follows from `Product Specification <nRF5340 Product Specification_>`_, Section 6.4.3.1, Table 22.
+   As a result, they cannot be used simultaneously.
+   For the SPI part of the nRF21540 interface to be functional, the UARTE0 node must be disabled in the network core's devicetree file.
```

Comment:
I believe `UARTE0` must be disabled for the app core as well. This also means that a different UART must be chosen for HCI to work in e.g. the `hci_uart` sample. The goes for all the other uses of `UARTE0` in the board devicetree and overlays.

Reply:
I disagree. nRF5340 network core has only one instance of `SPIM` peripheral: `SPIM0`. It's a separate instance than `SPIM0` on the application core so there's no need to modify the application core's devicetree file to avoid conflicts with network core's `SPIM0`. Please refer to nRF5340 PS v1.4 section 7.30.7 for details on the instances

Reply:
Yes, you are correct. I mixed them up. Just add a note to disable boot banner etc that by default uses `UARTE0` on network core, then.